main/src/ca/uwaterloo/flix/language/ast/KindedAst.scala

```diff
@@ -304,7 +304,7 @@ object KindedAst {
 
   case class Case(sym: Symbol.CaseSym, tpe: Type, sc: Scheme, loc: SourceLocation)
 
-  case class RestrictableCase(sym: Symbol.RestrictableCaseSym, tpe: Type, sc: Scheme, loc: SourceLocation)
+  case class RestrictableCase(sym: Symbol.RestrictableCaseSym, tpe: Type, sc: Scheme, sc2: Scheme, loc: SourceLocation)
```

Comment:
ignore  these things, i was just trying to carry both the new and the old schema around